catalyst-generator/src/main/java/com/intellecteu/catalyst/util/PluginBinding.java

```diff
@@ -0,0 +1,26 @@
+package com.intellecteu.catalyst.util;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Accordance between use cases and plugins needed
```

Comment:
This should be a part of YML config entry of /catalyst-service/src/main/resources/application.yml, alongside with other use-cases configuration. You can add your custom **mavenPlugins** section to describe plugin dependencies